File: DVLD_Project/Form5.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Diagnostics;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using BussnessLayerForDVLD;
using Microsoft.Win32;

namespace DVLD_Project
{
    public partial class Form5 : Form
    {
        public Form5()
        {
            InitializeComponent();
            
        }

        private void guna2Button1_Click(object sender, EventArgs e)
        {
            string txt = clsBussnesLayer.HasingPassWord(tbPassWord.Text);          
            int Num = clsBussnesLayer.LoginUser(tbUserName.Text,txt);
            clsBussnesLayer.SetLogin(Num);
            if (Num>0)
            {
                if (checkBox1.Checked)
                {
                    clsBussnesLayer.SetUserNameAndPassWord(tbUserName.Text.ToString(), tbPassWord.Text.ToString());
                }
                else
                {
                    clsBussnesLayer.SetUserNameAndPassWord("","");
                }
                this.Hide();
                frmMain form = new frmMain();
                form.ShowDialog();
            }
            else
            {
                string UsersEL = "Users_Logging_DVLD_Applcation";
                if (!EventLog.SourceExists(UsersEL))
                {
                    EventLog.CreateEventSource(UsersEL,"Applcation");
                   
                }
                EventLog.WriteEntry(UsersEL, $"This Login Is Denay For UserName : {tbUserName.Text} , PassWord : {tbPassWord.Text}", EventLogEntryType.Error);
                MessageBox.Show("nnnn");
            }
           
        }

        private void checkBox1_CheckedChanged(object sender, EventArgs e)
        {

        }

        private void Form5_Load(object sender, EventArgs e)
        {
            string[] strings = clsBussnesLayer.GetUserNameAndPassWord();
            if (strings!=null)
            {
                tbUserName.Text = strings[0];
                tbPassWord.Text = strings[1];
                checkBox1.Checked = true;
            }
        }
    }
}

File: DVLD_Project/Form5.vb
Imports System
Imports System.Data
Imports System.Windows.Forms
Imports BussnessLayerForDVLD

Namespace DVLD_Project
    Public Partial Class Form5
        Inherits Form
        Public Sub New()
            InitializeComponent()

        End Sub

        Private Sub guna2Button1_Click(sender As Object, e As EventArgs)
            Dim txt = ""
            For Each c In tbPassWord.Text.ToString()
                txt += Microsoft.VisualBasic.ChrW(Microsoft.VisualBasic.AscW(c) + 15)
            Next
            Dim Num = clsBussnesLayer.LoginUser(tbUserName.Text, txt)
            clsBussnesLayer.SetLogin(Num)
            If Num > 0 Then
                If checkBox1.Checked Then
                    clsBussnesLayer.UpdateUserCopy(0, 0, tbUserName.Text, tbPassWord.Text, True)
                Else
                    clsBussnesLayer.UpdateUserCopy(0, 0, "", "", True)
                End If
                Hide()
                Dim form As Form1 = New Form1()
                form.ShowDialog()
            Else
                MessageBox.Show("nnnn")
            End If

        End Sub

        Private Sub checkBox1_CheckedChanged(sender As Object, e As EventArgs)

        End Sub

        Private Sub Form5_Load(sender As Object, e As EventArgs)
            Dim dt As DataTable = clsBussnesLayer.FindUserCop()
            For Each dr As DataRow In dt.Rows
                tbUserName.Text = dr("UserName").ToString()
                tbPassWord.Text = dr("PassWord").ToString()
                checkBox1.Checked = True
                Exit For
            Next
        End Sub
    End Class
End Namespace
